src/state.ts

```diff
@@ -184,7 +184,7 @@ export async function initProjectDir(uri?: vscode.Uri): Promise<void> {
 export function resolvePath(filePath?: string): vscode.Uri {
   const root = getProjectWsFolder();
 
-  if (root.uri.scheme !== 'file') {
+  if (root && root.uri.scheme !== 'file') {
```

Comment:
@corasaurus-hex Should we get some kind of warnings about this from TS, that a property is being accessed on a potentially undefined value?